packages/cosmos-client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "cosmos-client",
+  "main": "index.ts",
```

Comment:
That wouldn't really work outside of our repo, but that's ok :)
we should be rather using `index.js` as `main` entry point. Providing types like `"types": "index.d.ts"`

Reply:
we'll go with "but that's ok" for now.